library/src/main/java/com/woxthebox/draglistview/BoardView.java

```diff
@@ -556,10 +560,20 @@ public void onDragStarted(int itemPosition, float x, float y) {
 
             @Override
             public void onDragging(int itemPosition, float x, float y) {
+                Integer column = getColumnOfList(recyclerView);
+                Integer row = itemPosition;
+                boolean positionChanged = !column.equals(mLastDragColumn) || !row.equals(mLastDragRow);
+                if (mBoardListener != null && positionChanged) {
+                    mLastDragColumn = column;
+                    mLastDragRow = row;
+                    mBoardListener.onDragItemChangedPosition(mDragStartColumn, mDragStartRow, getColumnOfList(recyclerView), row);
```

Comment:
Use "column" instead of calling the method again as it has already been done on line 563.